apps/studio/components/interfaces/DiskManagement/DiskManagementForm.tsx

```diff
@@ -310,6 +311,35 @@ export function DiskManagementForm() {
           />
           {!isFlyArchitecture && (
             <>
+              <div className="flex flex-col gap-y-3">
+                <DiskCountdownRadial />
+                {usedPercentage >= 90 && isWithinCooldownWindow && (
+                  <Admonition
+                    type="destructive"
+                    title="Database is currently at 90% of disk size"
```

Comment:
should be `is currently over 90%` right?

Reply:
oh yeap! good point ahha